internal/femto/femto.go

```diff
@@ -3,124 +3,145 @@ package femto
 
 import (
 	"encoding/json"
-	"errors"
 	"log/slog"
 	"net/http"
 	"sync/atomic"
+
+	"github.com/gpuctl/gpuctl/internal/types"
 )
 
 type Femto struct {
 	mux   http.ServeMux
 	reqNo atomic.Uint64
 }
 
-func (f *Femto) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	f.mux.ServeHTTP(w, r)
+type HTTPResponseContent[T any] struct {
```

Comment:
This name is way too long given that it's used everywhere. Can we call it `femto.Response`?

Reply:
We absolutely can. I have a bias for longer names for key API layers. We need to prioritise DX and code-clarity for layers that hide abstraction, for the same reason Go was selected.

Reply:
I'd say the oposite. Key API layers should have short, sweet names. Femto is all about HTTP, so it's redundant to have `HTTP` in the name. Similarly, we don't have a seperate ResponceMeta or ResponceBody to contend with, so why specify Content

(aside: Also if you wanted to return HTTP 204 No contend, but set some headers, you'd still use this, so not only is it too long, it's wrong)

I'm not sure if your saying your willing to do the rename here on not. But this current name is unacceptably long, and I'm not willing to merge it. See <https://journal.stuffwithstuff.com/2016/06/16/long-names-are-long/> for more.

Reply:
I think you've misread semantics there. Here Content = (Status, Header, Body), not Response = (Status, Header, Content). This is a case of isomoprhism up to naming, so we're really just deciding on ontology here. I wouldn't go so far as to claim its semantically "wrong". A response's content that has HTTP 204 is (204, ..., ()). 

If you're extremely keen on having your version of the naming, I'll get that done.

Reply:
Yes, please rename it. It's confusing, and too long to be so fundamental